assets/src/components/propertiesPanel/vehiclePropertiesPanel.tsx

```diff
@@ -198,15 +199,25 @@ const StatusContent = ({ selectedVehicle }: { selectedVehicle: Vehicle }) => (
   </>
 )
 
-const VehiclePropertiesPanel = ({ selectedVehicle }: Props) => {
-  const [tabMode, setTabMode] = useState<TabMode>("status")
+const VehiclePropertiesPanel = ({
+  selectedVehicle,
+  initialTab = "status",
+}: Props) => {
+  const [tabMode, setTabMode] = useState<TabMode>(initialTab)
 
   return (
     <div className="c-vehicle-properties-panel">
       <Header
         vehicle={selectedVehicle}
         tabMode={tabMode}
-        setTabMode={setTabMode}
+        setTabMode={(tab) => {
+          if (tab !== tabMode) {
```

Comment:
nitpick (non-blocking): I wasn't sure on initial read if `tab` and `tabMode` represent two different things. Maybe renaming `tab` to something like `newTabMode` or something otherwise containing `tabMode` could help? But I also like the brevity of just `tab` so really no need to change

Reply:
80dec4ea502324c3c9100909c5ea46177dca1caf